README.md

```diff
@@ -12,3 +11,25 @@ To run the tests, run this on your command line:
 ```
 ./phpunit.phar
 ```
+
+
+## Contributing
+
+Bug reports and pull requests are welcome on GitHub at https://github.com/indieweb/link_rel_parser-ruby/issues. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [IndieWebCamp Code of Conduct](https://indiewebcamp.com/code-of-conduct).
+
+1. Fork it
+2. Get it running
+3. Create your feature branch (`git checkout -b my-new-feature`)
+4. Write your code and **tests**
+5. Commit your changes (`git commit -am 'Add some feature'`)
+6. Push to the branch (`git push origin my-new-feature`)
+7. Create new Pull Request
+
+If you find bugs, have feature requests or questions, please
+[file an issue](https://github.com/indieweb/link-rel-parser-php/issues).
+
+
+## Code of Conduct
+
+Everyone interacting in the link-rel-parser-php codebase, issue tracker, chat room, and mailing lists is expected to follow the
+[link-rel-parser-php code of conduct](https://github.com/indieweb/link-rel-parser-php/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
might as well link to `[IndieWebCamp Code of Conduct](https://indiewebcamp.com/code-of-conduct)` here too